sdgym/benchmark.py

```diff
@@ -504,6 +624,15 @@ def benchmark_single_table(synthesizers=DEFAULT_SYNTHESIZERS, custom_synthesizer
         pandas.DataFrame:
             A table containing one row per synthesizer + dataset + metric.
     """
+    if run_on_ec2:
+        LOGGER.info("This will create an instance for the current AWS user's account.")
```

Comment:
In this case can we just straight up print the message so the user can't turn it off?

Reply:
Had to add the `noqa` comment to avoid linter picking it up